src/services/datahub-client.test.ts

```diff
@@ -378,20 +378,23 @@ describe('HttpDataHubClient', () => {
       );
       expect(response).toEqual({
         success: true,
-        value: [
-          {
-            name: '/email-marketing',
-            totalVisitors: 10122,
-            url: 'http://doppler.test/email-marketing',
-            withEmail: 400,
-          },
-          {
-            name: '/precios',
-            totalVisitors: 9000,
-            url: 'http://doppler.test/precios',
-            withEmail: 300,
-          },
-        ],
+        value: {
+          hasMorePages: false,
```

Comment:
Perhaps you could add a scenario for `hasMorePages=true`

Reply:
yes! i added now one